clusters/build-clusters/01_cluster/sshd-bastion/build.yaml

```diff
@@ -36,3 +36,45 @@ items:
         imageOptimizationPolicy: SkipLayers
     triggers:
     - type: ConfigChange
+
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    name: coredns
+    namespace: ci
+  spec:
+    lookupPolicy:
+      local: true
+
+- kind: BuildConfig
```

Comment:
All of this needs to be on `app.ci` to be on the `registry.ci.openshift.org` domain